site/app/templates/GlobalHeader.twig

```diff
@@ -180,7 +180,44 @@
                         {% endfor %}
                     </div>
                 </div>
+
+
+<div class="banner" id = "banner">
+{% set image_found = false %}
+
+  {% for imageData in imageDataArray %}
+      <div>
+      {% set imgName = "{{imageData.name}}" %}
+
+        {% if imageData.type != "link" %}
+            {% set extraStuff = "data:image/png;base64," ~ imageData.extra_info %}
+            <img id="club-banners_" class="club-banners {{imageData.name}}" src="data:imgName/png;base64,{{ imageData.data }}" onclick="newShowImage('{{extraStuff}}', '{{imageData.type}}' )" />
+        {% else %}
+            {% set extraStuff = imageData.extra_info %}
+            <a href="{{ extraStuff }}" target="_blank">
+            <img id="club-banners_" class="club-banners {{imageData.name}}" src="data:imgName/png;base64,{{ imageData.data }}" />
+            </a>
+        {% endif %}
+
+
+
+
+
+        <button class="delete-button" onclick="removeImage(this)">x</button>
+      </div>
+  {% endfor %}
+  <button class="prev-button" onclick="changeImage(-1)">&#10094;</button>
+  <button class="next-button" onclick="changeImage(1)">&#10095;</button>
```

Comment:
![image](https://github.com/Submitty/Submitty/assets/47994802/86192f7e-6445-492b-93ea-35defd8ea011)
You should update the CSS to make it look more like this, 
for next button change the border-radius value to `50% 0% 0% 50%`, and increase padding (in the screenshot I made it 0.5rem). the border-radius value for prev-button should be `0% 50% 50% 0%;`